MvcChatBot.Agent/Services/TwitchApiService.cs

```diff
@@ -17,14 +17,32 @@ public class TwitchApiService
         private TwitchAPI API;
         //private FollowerService FollowerService;
         private readonly TwitchSettings _settings;
-        private readonly HubConnection _connection;
+        private readonly HubConnection _connection;        
 
         public TwitchApiService(TwitchSettings settings, HubConnection connection)
         {
-            _settings = settings;
+            _settings = settings;           
+
+            API = new TwitchAPI();
+            API.Settings.ClientId = _settings.ClientId;
+            API.Settings.AccessToken = _settings.ChannelAuthToken;
+
             _connection = connection;
             _connection.StartAsync();
-            Task.Run(() => ConfigLiveMonitorAsync());
+            //Task.Run(() => ConfigLiveMonitorAsync());
+        }
+
+        public string GetCurrentStats()
```

Comment:
This method could be made async and return a Task<string> then just return await GetStatsAsync() :-)